appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -81,32 +88,50 @@ for ($i = 0; $i -lt $sarifLog.runs.Count; $i++){
     #Codeflow
     for($k = 0; $k -lt $sarifLog.runs[$i].results[$j].codeFlows.Count; $k++){
       for($l = 0; $l -lt $sarifLog.runs[$i].results[$j].codeFlows[$k].locations.Count; $l++){
-        $sarifLog.runs[$i].results[$j].codeFlows[$k].locations[$l].physicalLocation.uri = Rebase-Uri $sarifLog.runs[$i].results[$j].codeFlows[$k].locations[$l].physicalLocation.uri
+        $uri = $sarifLog.runs[$i].results[$j].codeFlows[$k].locations[$l].physicalLocation.uri
+        if($uri){
+          $sarifLog.runs[$i].results[$j].codeFlows[$k].locations[$l].physicalLocation.uri = Rebase-Uri $uri
+        }
       }
     }
 
     for($k = 0; $k -lt $sarifLog.runs[$i].results[$j].relatedLocations.Count; $k++){
-      $sarifLog.runs[$i].results[$j].relatedLocations[$k].physicalLocation.uri = Rebase-Uri $sarifLog.runs[$i].results[$j].relatedLocations[$k].physicalLocation.uri
+      $uri = $sarifLog.runs[$i].results[$j].relatedLocations[$k].physicalLocation.uri
+      if($uri){
+        $sarifLog.runs[$i].results[$j].relatedLocations[$k].physicalLocation.uri = Rebase-Uri $uri
+      }
     }
 
     for($k = 0; $k -lt $sarifLog.runs[$i].results[$j].stacks.Count; $k++){
-      $sarifLog.runs[$i].results[$j].stacks[$k].frames.uri = Rebase-Uri $sarifLog.runs[$i].results[$j].stacks[$k].frames.uri
+      $uri = $sarifLog.runs[$i].results[$j].stacks[$k].frames.uri
+      if($uri){
+        $sarifLog.runs[$i].results[$j].stacks[$k].frames.uri = Rebase-Uri $uri
+      }
     }
 
     for($k = 0; $k -lt $sarifLog.runs[$i].results[$j].fixes.Count; $k++){
-      $sarifLog.runs[$i].results[$j].fixes[$k].fileChanges.uri = Rebase-Uri $sarifLog.runs[$i].results[$j].fixes[$k].fileChanges.uri
+      $uri = $sarifLog.runs[$i].results[$j].fixes[$k].fileChanges.uri
+      if($uri){
+      $sarifLog.runs[$i].results[$j].fixes[$k].fileChanges.uri = Rebase-Uri $uri
+      }
     }
   }
 
   for($j=0; $j -lt $sarifLog.runs[$i].configurationNotifications.Count; $j++){
-    $sarifLog.runs[$i].configurationNotifications[$j].physicalLocation.uri = Rebase-Uri $sarifLog.runs[$i].configurationNotifications[$j].physicalLocation.uri
+    $uri = $sarifLog.runs[$i].configurationNotifications[$j].physicalLocation.uri
+    if($uri){
+      $sarifLog.runs[$i].configurationNotifications[$j].physicalLocation.uri = Rebase-Uri $uri
+    }
   }
 
   for($j=0; $j -lt $sarifLog.runs[$i].toolNotifications.Count; $j++){
-    $sarifLog.runs[$i].toolNotifications[$j].physicalLocation.uri = Rebase-Uri $sarifLog.runs[$i].toolNotifications[$j].physicalLocation.uri
+    $uri = $sarifLog.runs[$i].toolNotifications[$j].physicalLocation.uri
+    if($uri){
+      $sarifLog.runs[$i].toolNotifications[$j].physicalLocation.uri = Rebase-Uri $uri
+    }
   }
 }
 
 $newExtension = ".github.sarif"
 $writeLocation = [System.IO.Path]::ChangeExtension($logFilePath, $newExtension)
-$sarifLog | ConvertTo-Json -depth 999 | Out-File $writeLocation
+$sarifLog | ConvertTo-Json -Compress -depth 999 | Out-File $writeLocation
```

Comment:
This removes all white space and newlines. In general: favor readability over file size unless and until you have data demonstrating that file size is a problem. So revert this change.